docs/pages/index.js

```diff
@@ -96,7 +96,7 @@ const useStyles = makeStyles(theme => ({
 }));
 
 const GettingStartedLink = React.forwardRef((props, ref) => {
-  return <Link href="/getting-started/installation" naked prefetch ref={ref} {...props} />;
+  return <Link href="/getting-started/installation" naked ref={ref} {...props} />;
```

Comment:
For context, prefetch was added so speed up the page transition between the home and the next page. But maybe it's not really useful?

Reply:
It's deprecated because next handles this by default.

Reply:
We live in the future 😀